xarray/tests/test_variable.py

```diff
@@ -1508,8 +1508,8 @@ def test_reduce_funcs(self):
         assert_identical(v.all(dim='x'), Variable([], False))
 
         v = Variable('t', pd.date_range('2000-01-01', periods=3))
-        with pytest.raises(NotImplementedError):
-            v.argmax(skipna=True)
```

Comment:
Does anyone remember why we did not implement `nanargmax` for `pd.date_range`? It looks `np.nanargmax` takes care of this dtype.

Reply:
nope, no idea! :)